### ISSUE TITLE
Some content not rendering properly

### ISSUE THREAD

**Issue by kundan-ugale** [StaticMania/portio-hugo#17]
Cloned the repo and ran it locally. Everything works fine but some of the UI components are not rendering properly which is causing distorted UI for me. Tried on both chrome and firefox but same result.

pls check the attachment. Any suggestions?

![image](https://user-images.githubusercontent.com/59203525/95660823-ca918b00-0b2a-11eb-9a68-ff69362ce60a.png)

here's the console log from chrome browser

![image](https://user-images.githubusercontent.com/59203525/95660915-47bd0000-0b2b-11eb-9322-1547953500db.png)

**Comment by Error6251** [comment]
Which version are you using for hugo...? Type
 hugo version
And give us a screenshot...

**Comment by kundan-ugale** [comment]
Hugo v0.74.3, Windows 10 , Chrome Version 86.0.4240.75

**Comment by Error6251** [comment]
As far as I know, API key's are really vital for privacy. So developer doesn't use his API key on this project. For using the Google map you need to add your own Google Map API in [line 70] at config.toml . But your Bootstrap is breaking...

**Comment by kundan-ugale** [comment]
did you try locally, Is it working for you?

**Comment by Error6251** [comment]
Yes It worked fine on my computer with older version of Hugo at Aug 25, from my Windows 10, and it also working fine on my Ubuntu Linux with the newest version of Hugo. No problem here...

**Comment by musicislife08** [comment]
I have the same exact problem.  

- Create a new site with `hugo new site namehere`
- git clone the theme into themes/portio
- copy requested files into root
- run `hugo serve` and I have the same problem. 

I attempted to remove all of the instances calling the google maps api to see if thats what the issue was but I have been unable to figure this out.  

doesnt matter which version of hugo is used I tried 64 up to latest and all do the same thing on both windows and my ubuntu wsl instance

**Comment by Error6251** [comment]
![1](https://user-images.githubusercontent.com/46952682/96543881-2dbbb400-12c7-11eb-8e9d-f9b09ce3ee88.png)
![2](https://user-images.githubusercontent.com/46952682/96543893-344a2b80-12c7-11eb-8ead-3fdf02fdad59.png)


This are the commands I used for running this Hugo theme on my Ubuntu Linux 
I think Please be patent try again, follow this step by step...

- **hugo new site namehere**
- **cd namehere**
- **git init**
- **git submodule add git@github.com:StaticMania/portio-hugo.git themes/portio**
- Now go to the themes/portio/exampleSite
- Coppy everything (config.toml, content, data, resources, static) from this folder to root directory of this project(namehere)
- Now go to the root directory of this project(namehere), and run this command **hugo server** 


...If still you got any problem please inform here again...

**Comment by bcruik** [comment]
I'm also having the same issue :( 

I'm not a designer so not 100% sure but getting errors with `Uncaught TypeError: Cannot set property 'cssText' of undefined` in Chrome console when running from `hugo server`.

I'm on Windows 10, installed Hugo via chocolatey.

**Comment by cosmosisis** [comment]
Same issue, sometiong wrong with the .css file, I guess.

**Comment by hervyqa** [comment]
please use hugo extended.

**Comment by arpitaksoni** [comment]
I am using the following version.
Hugo Static Site Generator v0.79.0-1415EFDC/extended linux/amd64 BuildDate: 2020-11-27T09:16:17Z

The local server http://localhost:1313/ readers all the pages perfectly. The Github pages completely mess up the CSS and show the output as shown in the first post.

**Comment by Error6251** [comment]
The problem here is SCSS, we are not using CSS, Sooo.... Downloading the SCSS included Extended version of hugo...
If you have snap store or using Ubuntu write... 
`"snap install hugo --channel=extended"`
//Remember to remove old one//

**Comment by santiyounger** [comment]
Hey, thanks for the awesome theme. 
I have a similar situtation, 

When I do hugo server, everything looks great
it's only once I deploy it with Netlify 

That things in the css or scss break completly and only the contenet shows in aan ugly naked HTML style

did @Error6251 's solution work for anyone else, I was not aware of hugo extended version I'll try that next, thanks @Error6251

**Comment by arpitaksoni** [comment]
I had the extended version of hugo 0.79 and now v0.78.2. Still.. no difference.
The theme renders properly with http instead of https.
What do you mean by "Downloading the SCSS included Extended version of hugo"?

**Comment by Error6251** [comment]
![](https://user-images.githubusercontent.com/46952682/101736910-505f9180-3aee-11eb-8042-7be78ec5e240.png)

**Comment by arpitaksoni** [comment]
The issue does not get resolved by using the extended version only. The issue is browsers block some scripts. Somehow GitHub pages are considered as unauthenticated sources for scripts.

**Comment by santiyounger** [comment]
Hey @arpitaksoni thanks for reporting your solution.

I tried changing from `http` to `https` but it didn't work in my case.
> The issue does not get resolved by using the extended version only
Do you mean the extended version is needed as well?

Also, is it needed to add something like `netlify.toml` to make it work? I know you said you are using gitpages instead of Netlify, but do you have an equivalent to such file?

It would help a lot if you can outine any other possible steps that I might be missing, and in case your code is public, it would truly help if you link it, if it's private don't worry.

Thanks @arpitaksoni

**Comment by arpitaksoni** [comment]
@santiyounger 

1. Fork the theme repo in your account.  Add the fork as a submodule in your project. 
2. Generated website pages are in a separate repo with GH-pages enabled.

I am using the extended version of Hugo.

**Comment by santiyounger** [comment]
thanks you for the help @arpitaksoni  I will try

**Comment by cyn3rgy** [comment]
I had to make a couple of changes to my site which I've just built. I'm not sure if it was specific to me but maybe it will help someone. Firstly the google api key didn't work. I created a new API key and put it into the config.toml. I kept getting Development Purposes overlay on the map. Turns out v2 of the google map api is deprecated. I had to replace the js include with the v3 api code and then replace the code in footer partial with something google generated for me. THankfully their example targeted #map so it was pretty straight forward to just drop their code in and then replace the long and lat lines to use the existing code from the theme.

2ndly, I got a heap of CORS errors when deploying to netlify. This made style sheets and fonts not load. I also have a vanity domain (more on that later). To fix the CORS issue I added the following to the config.toml
```
[server]

  [[server.headers]]
  for = "/"

    [server.headers.values]
    X-Frame-Options = "DENY"
    X-XSS-Protection = "1; mode=block"
    X-Content-Type-Options = "nosniff"
    Referrer-Policy = "strict-origin-when-cross-origin"
    Feature-Policy = "accelerometer 'none'; camera 'none'; geolocation 'none'; gyroscope 'none'; magnetometer 'none'; microphone 'none'; payment 'none'; usb 'none'"
    Strict-Transport-Security = "max-age=31536000; includeSubDomains; preload"
    Content-Security-Policy = "default-src 'self'; script-src 'self' 'unsafe-inline' www.google-analytics.com maps.googleapis.com; img-src 'self' www.google-analytics.com; style-src 'self' 'unsafe-inline' fonts.googleapis.com; font-src 'self' fonts.gstatic.com fonts.googleapis.com; form-action 'self';"
```
I'm not that familiar with what it does but it seems to overcome the CORS issues that were showing in the chrome console.

The other change I did was replace most of the AbsURL with RelURL. This fixed a heap of problems I had with cross domain issues because my netlify site had more than 1 domain.

Hope it helps someone.

Andrew
